docs/getting-started/endpoint-management-req.asciidoc

```diff
@@ -1,4 +1,28 @@
 [[endpoint-management-req]]
 = Endpoint management requirements
```

Comment:
Only one small comment, the heading of this page feels a little bit misleading. We will add endpoint **management** feature privileges (hopefully in 8.7) but they aren't in there yet (I wouldn't call the response actions feature controls "endpoint management" related). Maybe we can make it something more generic/specific to feature controls, something like "Endpoint feature privileges"?

Reply:
@caitlinbetz Renaming the page sounds good. I just want to future-proof it, so it's accurate now but we won't need to rename the page later on.

Would "Endpoint feature privileges" still be a viable title once we add the proper _endpoint management_ feature privileges in ~8.7, plus any other privileges beyond that? It seems generic enough to me, but just want to make sure.

Reply:
@joepeeples I think so, but now that I'm thinking about it more, we should probably reference Defend instead of Endpoint. Maybe "Elastic Defend feature privileges" instead? ("...This allows you to use the principle of least privilege while managing access to Elastic Defend’s features.")

Reply:
"Elastic Defend feature privileges" sounds good to me! I'm also noticing that the [note](https://github.com/elastic/security-docs/pull/2763/files#diff-73a514535f8d3b85cb4377d7a92dd308132e918c6d9c0277acac460aa64840bfR8) in the intro could also be adjusted to be more general, less specific about "endpoint management". I'll touch up all this in a new commit.